src/gaurang/Assignment74.java

```diff
@@ -0,0 +1,42 @@
+/*return the list of all the prime numbers in a give list.*/
+
+package gaurang;
+
+import java.util.ArrayList;
+
+public class Assignment74 {
+
+	void getPrime(ArrayList<Integer> input) {
+		System.out.println("Given input : "+input);
+		for(int index=0 ; index<input.size() ; index++) {
+			int x = input.get(index);
+			if(x<=0) {
+				System.out.println("Invalid number.");
+			}else {
+				boolean flag = true;
+				for(int innerindex=2 ; innerindex <= x/2 ; innerindex++) {
```

Comment:
You can create separate method to verify is number is prime to optimize your code